sysdata/config/configdata.py

```diff
@@ -93,14 +95,23 @@ def add_single_element(self, element_name):
                 elements.append(element_name)
                 self._elements = elements
 
-    def get_element_or_missing_data(self, element_name):
-        result = getattr(self, element_name, missing_data)
+    def get_element(self, element_name):
+        try:
+            result = getattr(self, element_name)
+        except AttributeError:
+            raise missingData("Missing config element %s" % element_name)
         return result
 
-    def get_element_or_arg_not_supplied(self, element_name):
-        result = getattr(self, element_name, arg_not_supplied)
+    def get_element_or_default(self, element_name, default):
+        result = getattr(self, element_name, default)
         return result
 
+    def get_element_or_missing_data(self, element_name):
+        return self.get_element_or_default(element_name, missing_data)
+
+    def get_element_or_arg_not_supplied(self, element_name):
```

Comment:
Yes, I expect both of those methods to go away eventually.

I considered adding a DeprecationWarning, but considering how widely they are used, I thought it would be too annoying to see those messages all the time.